network/src/network.rs

```diff
@@ -938,6 +938,7 @@ impl NetworkService {
                 let mut runtime = runtime::Builder::new()
                     .core_threads(num_threads)
                     .name_prefix("NetworkRuntime-")
+                    .panic_handler(|_| warn!("One future task panic"))
```

Comment:
See `util/logger/src/lib.rs` how to print the backtrace.

Reply:
I think we can expose it as a global pub function. Given panic info, print the backtrace via log.